llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -196,11 +198,13 @@ class YkControlPoint : public ModulePass {
         Intrinsic::getDeclaration(&M, Intrinsic::frameaddress, {VoidPtr});
     Value *FAddr = Builder.CreateCall(FrameAddress, {Builder.getInt32(0)});
 
-    // Insert call to the new control point.
+    // Insert call to the new control point. The last argument is the stackmap
+    // id belonging to the control point. This is temporarily set to INT_MAX
```

Comment:
I am not sure what value `UINT_MAX` represents. `UINT64_MAX` is the maximum value for a `uint64_t`, but looking below I see we do `getInt64` and I'm not sure if that takes a signed or unsigned int (I guess the latter).

Reply:
Oh right. Yeah this should be `UINT64_MAX`.

Reply:
Fixed in [a6357e6](https://github.com/ykjit/ykllvm/pull/182/commits/a6357e61b18e9a3142bade26fb8fc5f7ad10a637)